test/pytest/test_code.py

```diff
@@ -26,6 +26,8 @@
     ['mypy', '--no-incremental', 'src/cockpit'],
 
     ['ruff', 'check', '--no-cache', '.'],
+
```

Comment:
Another unnecessary newline, this makes the declaration appear oddly disconnected.